composer.json

```diff
@@ -78,7 +78,7 @@
     "config": {
         "bin-dir": "bin",
         "platform": {
-            "php": "7.2.9"
+            "php": "7.3"
```

Comment:
is that correct? I think it needs to be a specific version

Reply:
Composer doesn't complain about it:
```
$ composer validate
./composer.json is valid
```

Should I adjust it to `7.3.0`?

Reply:
No it's more convenient if that just works!